scripts/py_matter_idl/matter_idl/generators/registry.py

```diff
@@ -36,6 +37,14 @@ def Create(self, *args, **kargs):
             return BridgeGenerator(*args, **kargs)
         elif self == CodeGenerator.CPP_APPLICATION:
             return CppApplicationGenerator(*args, **kargs)
+        elif self == CodeGenerator.CUSTOM:
+            # Use a package naming convention to find the custom generator:
+            # ./matter_idl_plugin/__init__.py defines a subclass of CodeGenerator named CustomGenerator.
+            # The plugin is expected to be in the path provided via the `--plugin <path>` cli argument.
+            # Replaces `from plugin_module import CustomGenerator``
+            plugin_module = __import__(kargs['plugin_module'])
```

Comment:
Just wondering, https://docs.python.org/3/library/importlib.html#importlib.__import__ seems to encourage using `importlib.import_module`  should we do that?

Reply:
changed to importlib